rails/app/models/speaker.rb

```diff
@@ -1,7 +1,24 @@
+require 'pry'
+# == Schema Information ==
```

Comment:
Did you mean to delete `require 'pry'`? 😸

Reply:
Whoops, yes!